Casks/p/piclist.rb

```diff
@@ -1,9 +1,9 @@
 cask "piclist" do
   arch arm: "arm64", intel: "x64"
 
-  version "2.9.7"
-  sha256 arm:   "4bc609bc7eac921be33baae19d220ef99989b9e07984eaebee02427a029fa132",
-         intel: "874d9334885bbc721a5a1666cff412334787917eaa649e829298cfac4cdbcd67"
+  version "2.9.8"
+  sha256 arm:   "1b3f8ffe24774e584c9c07bc4807bf53f49dcb6809befa3a15c602158db3c448",
+         intel: "2fa2c9d8f0f61d26fb232aa1f64cf27314843e8a018f65dea01f2ff74274d0b2"
 
   url "https://release.piclist.cn/latest/PicList-#{version}-#{arch}.dmg"
```

Comment:
```suggestion
  url "https://github.com/Kuingsmile/PicList/releases/download/v#{version}/PicList-#{version}-#{arch}.dmg",
        verified: "github.com/Kuingsmile/PicList/"
```

Can we use the GitHub download instead?